docs/deployment.md

```diff
@@ -17,63 +17,59 @@ Inhalt:
+## Table of Contents
+{: .no_toc }
+1. TOC
+{:toc}
 
-## Install a plug-in
-The generated **pip** file can be installed directly in the Auto Importer as a plug-in.\
-This works via the install function in the Auto Importer, via drag & drop into the Auto Importer window and via file system.
+## Creating a plug-in package
+A *PiWeb Installable Plug-in* (.pip) package file is automatically created on every build of your plug-in as long as the *Import SDK* nuget is referenced by the project. The file is written to `$(PackageOutputPath)` which usually is (depending on your build configuration) either `bin\Debug` or `bin\Release`. A .pip file contains a the complete plug-in implementation and does not have any other dependencies. As a single file, it is easy to distribute.
 
-### Download compiled example plug-in
+{: .note}
+>You may switch building the package on and off by setting the `GeneratePluginPackageOnBuild` property in the project file:
+>```xml
+><GeneratePluginPackageOnBuild>true</GeneratePluginPackageOnBuild>
+>```
+
+## Distributing a plug-in package
+Unfortunately, there is currently no package management infrastructure available that could be used to manage, share and distribute packages centrally. You can, however, manage and distribute packages by your own means.
+
+## Installing a plug-in package
+Plug-in packages can directly be installed in *PiWeb Auto Importer* in two different ways:
+1. by dropping the package file to install anywhere in *PiWeb Auto Importer*, or
+1. by opening the plug-in management view via `File > Plug-ins...` and then clicking on `Install plug-in...` in the management actions dropdown. You will then be prompted to choose the package file to install.
+   
+   ![Install plug-in](../assets/images/deployment/ai_install_1.png "Install plug-in")
+
+*PiWeb Auto Importer* will show an overview of the plug-in to install. You can continue by pressing the `Install` button.
+
+![Install window](../assets/images/deployment/ai_install_2.png "Install window")
+
+Now the new plug-in is listed in the plug-in management view. However, it is not active yet since a restart is required to actually install and load it. At this point you can use the opportunity to install or remove further plug-ins before finally pressing the `Restart now` button in the info belt at the bottom of the window.
+
+![Restart request](../assets/images/deployment/ai_install_3.png "Restart request")
+
+Immediately after the restarting, the plug-in installer opens. It displays all pending installation operations. You can press the `Run now` button to apply all these changes. At this point the installer prompts for administrative privileges if you do not already have them, stops all import plans running as windows services and applies the plug-in changes. Afterwards, all previously stopped windows services will be started again. When everything is done, you can close the window and *PiWeb Auto Importer* starts normally.

```

Comment:
"the restarting" sounds strange. Maybe:
The plug-in installer opens immediately after the restart.

Reply:
Rewritten.